src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Batch.cs

```diff
@@ -429,6 +429,8 @@ private async Task ExecuteOnce(DbConnection conn, CancellationToken cancellation
                         {
                             do
                             {
+                                // Verify that the cancellation token hasn't been canceled
```

Comment:
nit: I think the code is self-documenting in this case, but that may just be me 😄